Source/WebCore/animation/TimelineRange.h

```diff
@@ -29,16 +29,30 @@
 
 namespace WebCore {
 
+enum class TimelineRangeStartOrEnd : bool { Start, End };
+
 struct SingleTimelineRange {
     enum class Name { Normal, Omitted, Cover, Contain, Entry, Exit, EntryCrossing, ExitCrossing };
 
     Name name { Name::Normal };
     Length offset;
+
+    bool operator==(const SingleTimelineRange& other) const = default;
+
+    static bool isDefault(const Length&, TimelineRangeStartOrEnd);
+    static bool isDefault(const CSSPrimitiveValue&, TimelineRangeStartOrEnd);
+
+    static bool isOffsetValue(const CSSPrimitiveValue&);
+
+    static Name timelineName(CSSValueID);
```

Comment:
I know you made the function name shorter but just for the record, I still prefer the single word `name`.

Reply:
This was to avoid clashing with the member name.